src/Sarif.Multitool.Library/ExportValidationRulesMetadataOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("export-validation-rules", HelpText = "Export rules metadata to a SARIF or SonarQube XML file.")]
```

Comment:
>or SonarQube XML file [](start = 82, length = 21)

Do we really do that?

Reply:
@michaelcfanning, do you know if this exports a SonarQube XML? I saw in the code that we export XML, but I'm not sure if that is a simple XML or not.

---
In reply to: [489726571](https://github.com/microsoft/sarif-sdk/pull/2073#discussion_r489726571) [](ancestors = 489726571)

Reply:
Yes it does. Or it did, successfully, several years ago when originally authored. I don't know if the format has kept pace with current SonarQube requirements.

---
In reply to: [489730440](https://github.com/microsoft/sarif-sdk/pull/2073#discussion_r489730440) [](ancestors = 489730440,489726571)